### PR TITLE
Update of sapor config map and values.yml - including ldap & x509 con…

### PR DESCRIPTION
**Pull request by girichinna27**
…figs

### CODE REVIEW

File: charts/oes/config/oes-sapor/application.yml [hunk 1]
```diff
@@ -4,14 +4,19 @@ oes:
...
-  {{- if .Values.sapor.config.spinnaker.spinAdminLoginEnabled }}
-  adminUsername: {{ .Values.sapor.config.spinnaker.spinAdminUsername }}
-  adminPassword: {{ .Values.sapor.config.spinnaker.spinAdminPassword }}
-  {{- end }}
+  ldap:
+    enabled: {{ .Values.sapor.config.spinnaker.ldap.ldapEnabled }}
+    username: {{ .Values.sapor.config.spinnaker.ldap.ldapUsername }}
+    password: {{ .Values.sapor.config.spinnaker.ldap.ldapPassword }}
+    adminLoginEnabled: {{ .Values.sapor.config.spinnaker.ldap.ldapAdminLoginEnabled }}
+    adminUsername:
```
Comment (abhinayopsmx) [inline charts/oes/config/oes-sapor/application.yml:13]: {{ .Values.sapor.config.spinnaker.spinAdminUsername }}

It should have this value

File: charts/oes/config/oes-sapor/application.yml [hunk 2]
```diff
@@ -4,14 +4,19 @@ oes:
...
+    adminPassword:
```
Comment (abhinayopsmx) [inline charts/oes/config/oes-sapor/application.yml:14]: {{ .Values.sapor.config.spinnaker.spinAdminPassword }}

It should have this value

File: charts/oes/values.yaml [hunk 3]
```diff
@@ -281,6 +281,22 @@ sapor:
...
+        ldapPassword: opsmx@123
+        ldapAdminLoginEnabled: false
+        ldapAdminUsername:
+        ldapAdminPassword:
+
+      ## X509
+      x509:
+        enabled: false
+        client:
+          file: /opt/opsmx/client.p12
```
Comment (abhinayopsmx) [inline charts/oes/values.yaml:297]: How is this certificate mounted on a sapor pod? We should have a config map that will be mounted onto the sapor pod if x509 is enabled.

**Review by abhinayopsmx — CHANGES_REQUESTED** [review]

File: charts/oes/values.yaml [hunk 4]
```diff
@@ -281,6 +281,22 @@ sapor:
       spinAdminUsername: admin
```
Comment (abhinayopsmx) [inline charts/oes/values.yaml:281]: We should remove these as they are moved under ldap now.

File: charts/oes/values.yaml [hunk 5]
```diff
@@ -281,6 +281,22 @@ sapor:
...
       spinAdminPassword: admin
 
+      ## LDAP
+      ldap:
+        ldapEnabled: true
+        ldapUsername: admin
+        ldapPassword: opsmx@123
+        ldapAdminLoginEnabled: false
+        ldapAdminUsername:
+        ldapAdminPassword:
```
Comment (abhinayopsmx) [inline charts/oes/values.yaml:291]: We should have some dummy values for these as helm doesn't allow empty value substitution

**Review by abhinayopsmx — CHANGES_REQUESTED** [review]

File: charts/oes/values.yaml (hunk 5 above)
Comment (girichinna27) [reply charts/oes/values.yaml:291]: I put this dummy values for username & passwords:
```
  ldapAdminUsername: admin
  ldapAdminPassword: admin
```

File: charts/oes/values.yaml (hunk 4 above)
Comment (girichinna27) [reply charts/oes/values.yaml:281]: Ok

File: charts/oes/config/oes-sapor/application.yml (hunk 1 above)
Comment (girichinna27) [reply charts/oes/config/oes-sapor/application.yml:13]: Updated

File: charts/oes/config/oes-sapor/application.yml (hunk 2 above)
Comment (girichinna27) [reply charts/oes/config/oes-sapor/application.yml:14]: Updated

File: charts/oes/values.yaml (hunk 3 above)
Comment (girichinna27) [reply charts/oes/values.yaml:297]: Created new config map and enabled the mount path to sapor-x509 config map

**Pull request merged**